libopenage/renderer/opengl/shader_program.cpp

```diff
@@ -204,15 +206,6 @@ GlShaderProgram::GlShaderProgram(const std::vector<resources::ShaderSource> &src
 void GlShaderProgram::use() const {
 	glUseProgram(*this->handle);
 
-	for (auto const &pair : this->textures_per_texunits) {
```

Comment:
Why remove this?

Reply:
I made a new system for managing textures which can be found in `texturemanager.cpp`. So this was not compatible with it. But this can be added back and a workaround can be found. I thought the system for managing various textures and texture units was a little too closed for optimisation.

Reply:
Ok, i'll look at that - if it's indeed better, there's no need to revert back to the old code.